pages/common/verilator.md

```diff
@@ -0,0 +1,20 @@
+# verilator
+
+> Converts Verilog and SystemVerilog hardware description language (HDL) designs into a C++ or SystemC model that after compiling can be executed.
+> More information: <https://veripool.org/guide/latest/>.
+
+- Build the C project in the current directory:
+
+`verilator --binary -j 0 -Wall {{path/to/source.v}}`
```

Comment:
```suggestion
`verilator --binary --build-jobs 0 -Wall {{path/to/source.v}}`
```

Optional change: `-j` alias can be used: long options are more readable are memorable.

```suggestion
`verilator --binary -j 0 -Wall {{path/to/source1.v path/to/source2.v ...}}`
```

Optional change: several source files can be passed.


```suggestion
`verilator --binary --build-jobs 0 -Wall {{path/to/source1.v path/to/source2.v ...}}`
```

Optional change: two fixes can be applied together.